dali/test/python/test_dali_tf_dataset_pipelines.py

```diff
@@ -33,20 +29,15 @@ def get_min_shape_helper(batch, max_shape):
     `batch` can also be a string "dataset" that indicates we passed a Dataset object as input
     without specifying the batch mode through: Input(dataset, batch=...)
     """
-    if batch is None or batch == True:
+    if batch is None or batch:
```

Comment:
This changes the semantics of the function.

>`batch` can also be a string "dataset"

```
if "dataset":
    takes this branch
```  

```
if "dataset" == True:
    does not take this branch
```

Reply:
Done